docs/01-index.md

```diff
@@ -0,0 +1,29 @@
+<img src="{{ path '/images/design-cover-img@2x.jpg' }}" alt="Welcome to design.stjude.cloud!" width="1280px" height="320px" style="max-width: unset"/>
+
+## Getting Started
+1. If you need to mockup or prototype, you can start by downloading our Sketch DSM - <a href="https://www.sketch.com/s/5f06f430-fa20-4aaf-ad60-6898bbda3de3">Sketch Design System</a>
+
+2. If you are using the theme in your project or prototype simply install the <a href="https://github.com/stjudecloud/theme" target="_blank">theme NPM packages</a>:
+
+  * https://github.com/stjudecloud/theme
+  * https://github.com/stjudecloud/theme/packages/140741
+
+3. We use <a href="https://getbootstrap.com/" target="_blank">Bootstrap</a> as the base for the St. Jude Cloud theme.
+
+4. We use <a href="https://fractal.build/" target="_blank">Fractal</a> for design.stjude.cloud:
+
+## Setup & Install
+1. ```git clone https://github.com/stjudecloud/design-system.git```
+2. ```npm install```
+3. ```npm i -g @frctl/fractal```
+4. Make your changes
+5. Compile SASS: ```npm run css-compile```
+6. Lint your changes: ```npm run css-lint```
+7. ```npm run watch``` is also provided for continuous compilation during development.
+8. ```fractal start --sync```
+
+## Contributing
+You do not need to commit changes to ```dist/css/*``` or ```guide/css/*``` as part of your pull request. These directories will be updated on release.
```

Comment:
I think we can now remove references to guide, no?

Reply:
Yeah, thanks. I also updated the README.